lib/pages/user_action/create/create_user_action_form_step2.dart

```diff
@@ -41,33 +55,66 @@ class CreateUserActionFormStep2 extends StatelessWidget {
+              onSelected: (value) {
+                widget.onTitleChanged(value);
+                Future.delayed(AnimationDurations.fast, () {
+                  descriptionFocusNode.requestFocus();
+                  _scrollToDescription(context);
+                });
+              },
+              actionType: widget.actionType,
             ),
-            if (viewModel.titleSource.isFromUserInput) ...[
+            if (widget.viewModel.titleSource.isFromUserInput) ...[
               const SizedBox(height: Margins.spacing_m),
               Text(Strings.userActionTitleTextfieldStep2, style: TextStyles.textBaseBold),
               const SizedBox(height: Margins.spacing_s),
               BaseTextField(
-                initialValue: viewModel.titleSource.title,
-                onChanged: (value) => onTitleChanged(CreateActionTitleFromUserInput(value)),
+                initialValue: widget.viewModel.titleSource.title,
+                onChanged: (value) => widget.onTitleChanged(CreateActionTitleFromUserInput(value)),
               ),
             ],
             const SizedBox(height: Margins.spacing_m),
-            Text(
-              Strings.userActionDescriptionTextfieldStep2,
-              key: descriptionKey,
-              style: TextStyles.textBaseBold,
-            ),
-            const SizedBox(height: Margins.spacing_s),
-            BaseTextField(
-              initialValue: viewModel.description,
-              maxLines: 5,
-              onChanged: (value) {
-                onDescriptionChanged(value);
-                _scrollToDescription(context);
-              },
+            AnimatedSwitcher(
+              duration: AnimationDurations.fast,
+              child: widget.viewModel.titleSource.isNone
+                  ? SizedBox.shrink()
+                  : Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Text(
+                          Strings.userActionDescriptionTextfieldStep2,
+                          key: descriptionKey,
+                          style: TextStyles.textBaseBold,
+                        ),
+                        const SizedBox(height: Margins.spacing_s),
+                        Text(
+                          Strings.userActionDescriptionDescriptionfieldStep2,
+                          style: TextStyles.textSRegular(),
+                        ),
+                        const SizedBox(height: Margins.spacing_s),
+                        const SizedBox(height: Margins.spacing_s),
+                        BaseTextField(
+                          focusNode: descriptionFocusNode,
+                          controller: descriptionController,
+                          hintText: Strings.exampleHint + widget.viewModel.titleSource.descriptionHint,
+                          maxLines: 5,
+                          suffixIcon: descriptionController.text.isEmpty
```

Comment:
En Nice to have, est-ce que tu pourrais pas faire en sorte que la croix reste ferrée en haut à droite du TextField si c'est pas trop relou ?